op-chain-ops/deployer/state/deploy_config.go

```diff
@@ -65,6 +65,13 @@ func DefaultDeployConfig() genesis.DeployConfig {
 				SystemConfigStartBlock:    0,
 			},
 		},
+		FaultProofDeployConfig: genesis.FaultProofDeployConfig{
+			FaultGameWithdrawalDelay:        604800,
+			PreimageOracleMinProposalSize:   126000,
+			PreimageOracleChallengePeriod:   86400,
+			ProofMaturityDelaySeconds:       604800,
+			DisputeGameFinalityDelaySeconds: 302400,
+		},
```

Comment:
What's the difference between these values and the ones in `op-chain-ops/deployer/bootstrap/bootstrap.go`?

Reply:
I'm planning on integrating them together as part of a later PR. Users should be able to override these but right now they can't.